Tests/PointFreeTests/TestCase.swift

```diff
@@ -9,11 +10,9 @@ class TestCase: XCTestCase {
 
     AppEnvironment.push(
       env: .init(
-        airtableStuff: mockCreateRow(result: .right(unit)),
-        fetchAuthToken: mockFetchAuthToken(result: .right(.init(accessToken: "deadbeef"))),
-        fetchGitHubUser: mockFetchGithubUser(
-          result: .right(.init(email: "hello@pointfree.co", id: 1, name: "Blob"))
-        )
+        airtableStuff: const(const(pure(unit))),
```

Comment:
lol, `mockCreateRow` was curried, so gotta `const >>> const` to stub it out. whatev, we will prob find better ways for all this some day.